### PR TITLE
Added Accrualify section

### PR DESCRIPTION
**Pull request by ialorro**
How to use Accrualify section

### CODE REVIEW

**Review by TQuock — DISMISSED** [review]

File: operations/finance/spending-company-money/how-to-use-accrualify/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## What is Accrualify?
+
+As of December 1, 2020, the Finance team have implemented a new Procure-to-Pay automation software called Accrualify for the Procurement and Accounts Payable function. With this, you teams will have visibility into invoice status, approval workflows will be automated, and budgeting will be more streamlined. The purpose of this section is to provide step-by-step instructions on how to use Accrualify.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/README.md]:
```suggestion
As of December 1, 2020, the Finance team has implemented a new Procure-to-Pay automation system called Accrualify for Procurement and Accounts Payable workflows. With Accrualify, your teams will have visibility into invoice status, approval workflows will be automated, and budgeting will be more streamlined. The purpose of this section is to provide step-by-step instructions on how to use Accrualify.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-set-up-your-account.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## How to set up your account
+
+The Finance team will send you a Welcome email from Accrualify if you need to do any of the following:
+
+* Submit a Purchase Order
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-set-up-your-account.md]:
```suggestion
* Submit a purchase order
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-set-up-your-account.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+* Review a Purchase Order
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-set-up-your-account.md]:
```suggestion
* Review a purchase order
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-set-up-your-account.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+* Review a vendor invoice
+
+If you need to submit a purchase order, please make sure you've submitted a budget request. Once your [budget request](https://handbook.mattermost.com/operations/finance/spending-company-money/budget) has been approved, the Finance team will provision you on Accrualify. You should receive a “Welcome to Accrualify” email, which includes a link to set up your account.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-set-up-your-account.md]:
```suggestion
If you need to submit a purchase order, please make sure you've submitted a [budget request](https://handbook.mattermost.com/operations/finance/spending-company-money/budget). Once your budget request has been approved, the Finance team will provision you an account on Accrualify. You should receive a “Welcome to Accrualify” email, which includes a link to set up your account.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-set-up-your-account.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+![](../../../../.gitbook/assets/How-to-set-up-your-Accrualify-account-1.PNG)
+
+Once you have set up your account, you will be able to view your Dashboard. From your Dashboard, you will be able to view all your Actionable Items.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-set-up-your-account.md]:
```suggestion
Once you have set up your account, you can log in to see your Accrualify dashboard to review and take action on your actionable items.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+## How to navigate Accrualify
+
+When you log into Accrualify, you'll see your dashboard. On the upper right hand corner, you can see your avatar.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md]:
```suggestion
When you log in to Accrualify, you're taken directly to your dashboard. In the upper right hand corner of the screen, you can see your avatar.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+![](../../../../.gitbook/assets/How-to-navigate-Accrualify-1.PNG)
+
+From here you can update your profile and access the Accrualify user manual.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md]:
```suggestion
Select your avatar to update your user profile and to access the Accrualify user manual.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+![](../../../../.gitbook/assets/How-to-navigate-Accrualify-2.PNG)
+
+On the left hand side, you'll see the different modules you have access to and can toggle between.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md]:
```suggestion
In the left hand navigation pane you'll see the different modules you can access. Some of the common modules you will use are described below.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+![](../../../../.gitbook/assets/How-to-navigate-Accrualify-3.PNG)
+
+In the **Vendor** module, you'll see the list of vendors that Mattermost purchases from. To search for vendors, you can type the name of the vendor in the text box. To add columns for other information, click on the green plus sign to bring down the columns menu. From here you are able to add additional columns on the grid, such as **ID**, **Account manager**, and **Terms**.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md]:
```suggestion
Select the **Vendor** module to see the list of vendors that Mattermost purchases from. 
- To search for a vendor, enter the name of the vendor in the **Vendor** text box. 
- To display more vendor details, select the green plus sign to the far right of the screen to see available column options. From here, select the columns to display on the grid, such as **Id**, **Acct Mgr**, and **Terms**.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+![](../../../../.gitbook/assets/How-to-navigate-Accrualify-4.PNG)
+
+![](../../../../.gitbook/assets/How-to-navigate-Accrualify-5.PNG)
+
+In the **Purchases** module, you'll see the list of purchase orders that you have submitted.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md]:
```suggestion
Select the **Purchases** module to see purchase orders that you've submitted.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+
+![](../../../../.gitbook/assets/How-to-navigate-Accrualify-6.PNG)
+
+Within the **Purchases** module, you'll see the list of purchase orders that are on your queue for review and approval.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md]:
```suggestion
Within the **Purchases** module, you'll also see purchase orders ready for your review and approval.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+
+![](../../../../.gitbook/assets/How-to-navigate-Accrualify-7.PNG)
+
+In the **Invoices** module, you will see all the invoices that are on your queue for approval.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md]:
```suggestion
Select the **Invoices** module to see invoices that are ready for your approval.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+
+![](../../../../.gitbook/assets/How-to-navigate-Accrualify-8.PNG)
+
+In the **Credit memos** module, you will also be able to see any credit memos that would need to be applied to an invoice. This would probably not happen too often.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md]:
```suggestion
Select the **Credit Memos** module to see any credit memos that need to be applied to an invoice. This is something that you'll do infrequently.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+
+![](../../../../.gitbook/assets/How-to-navigate-Accrualify-9.PNG)
+
+In the **Payments** module, you'll see the payment status of your invoices.
+
+![](../../../../.gitbook/assets/How-to-navigate-Accrualify-10.PNG)
+
+At this point in time, you will not need to use the Accruals module and the Corporate Cards module. For Corporate Card requests, please refer to the [“Virtual Card”](https://handbook.mattermost.com/operations/finance/spending-company-money/procurement/request-a-divvy-virtual-credit-card) section of the Handbook.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-navigate-accrualify.md]:
```suggestion
For now, you won't need to access the **Accruals** or **Corporate Cards** modules.

For Corporate Card requests, please refer to the [Virtual Credit Card](https://handbook.mattermost.com/operations/finance/spending-company-money/procurement/request-a-divvy-virtual-credit-card) section of the Handbook.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md [hunk 15]
```diff
@@ -0,0 +1,28 @@
+## How to Review a Vendor Invoice
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md]:
```suggestion
## How to Review a vendor invoice
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+
+If you need to review an invoice, Accrualify will send you an email alerting you that an invoice is on your queue for review and approval. The email will include the link “Review Invoice Request”, which will direct you to the invoice that needs your attention.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md]:
```suggestion
If you need to review a vendor invoice, Accrualify will send you an email alerting you that an invoice is ready for review and approval. From the email, select the “Review Invoice Request” link to go directly to the invoice that needs your attention.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md [hunk 17]
```diff
@@ -0,0 +1,28 @@
...
+
+![](../../../../.gitbook/assets/How-to-review-a-vendor-invoice-1.PNG)
+
+Once you're directed to the pending invoice, please review the following details:
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md]:
```suggestion
Once you're directed to the pending vendor invoice, please review the following details:
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md [hunk 18]
```diff
@@ -0,0 +1,28 @@
...
+
+1. **Vendor:** Please make sure you are reviewing an invoice for the vendor that you have submitted a PO for.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md]:
```suggestion
1. **Vendor:** Make sure you're reviewing a vendor invoice that has a submitted purchase order.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md [hunk 19]
```diff
@@ -0,0 +1,28 @@
...
+2. **Amount:** Please make sure the amount on the invoice matches the PO submitted.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md]:
```suggestion
2. **Amount:** Make sure the amount on the vendor invoice matches the submitted purchase order.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md [hunk 20]
```diff
@@ -0,0 +1,28 @@
...
+3. **Department:** Please make sure the department this spend is going to is your department or is properly allocated to your department if the spend is shared with other departments.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md]:
```suggestion
3. **Department:** Make sure this spend is for your department (or is properly allocated to your department in cases where the spend is shared with other departments).
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md [hunk 21]
```diff
@@ -0,0 +1,28 @@
...
+4. **Description:** Please make sure the description on the invoice is accurate (for example, it describes the service performed by the vendor for Mattermost).
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md]:
```suggestion
4. **Description:** Make sure the description on the invoice is accurate. For example, a description of the service performed by the vendor for Mattermost.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md [hunk 22]
```diff
@@ -0,0 +1,28 @@
...
+
+![](../../../../.gitbook/assets/How-to-review-a-vendor-invoice-2.PNG)
+
+Once you have reviewed the invoice and are ready to approve, click **Approve and Next**. After you approve the invoice, a Confirmation will pop up to ask **Are you sure you want to approve (invoice xxx)?**. Click **Ok**.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md]:
```suggestion
Once you have reviewed the vendor invoice and are ready to approve it, select **Approve and Next**. 
When prompted to confirm the approval, select **OK**.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md [hunk 23]
```diff
@@ -0,0 +1,28 @@
...
+
+![](../../../../.gitbook/assets/How-to-review-a-vendor-invoice-3.PNG)
+
+If you need to reject the invoice, click **Reject and Next**. If you reject the invoice, it means you disagree with one or more items on the invoice. When you reject the invoice, please provide the reasons for the reject in the free text box that appears after you click on **Reject and Next**.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md]:
```suggestion
If you need to reject the vendor invoice, select **Reject and Next**. Rejecting the invoice means you disagree with one or more invoice details. 

When prompted, please provide the reasons for rejecting the invoice in the **Rejection Comments** text field then select **OK**.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md [hunk 24]
```diff
@@ -0,0 +1,28 @@
...
+
+![](../../../../.gitbook/assets/How-to-review-a-vendor-invoice-4.PNG)
+
+To check for other open items on your dashboard, click on **Dashboard** on the left hand side of the screen. From the Dashboard, you will see other Actionable Items.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-vendor-invoice.md]:
```suggestion
, Select **Dashboard** to return to your dashboard and check for other open items that need your attention.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/frequently-asked-questions.md [hunk 25]
```diff
@@ -0,0 +1,21 @@
+## FAQs
+
+**I had an account with Bill.com, do I need to deprovision myself from Bill.com?**
+
+* Nope! The Finance team will deprovision your account for you. You should stop receiving emails from Bill.com by January 31, 2021.
+
+**How do I set up my account on Accrualify?**
+
+* If you need to submit a PO, review a PO, and/or review an invoice, the Finance team will provision you on Accrualify. You'll receive a Welcome email that will include a link to set up your account. Please note that not all staff members will be provisioned on Accrualify, so if you feel that you need access, please contact Procurement@mattermost.com.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/frequently-asked-questions.md]:
```suggestion
* If you need to submit a purchase order, review a purchase order, and/or review a vendor invoice, the Finance team will provision you an account on Accrualify. You'll receive a Welcome email that includes a link to set up your account. Please note that not all staff members will be provisioned accounts on Accrualify, so if you feel that you need access, please contact Procurement@mattermost.com.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/frequently-asked-questions.md [hunk 26]
```diff
@@ -0,0 +1,21 @@
...
+
+**Where can I find my to-do items on Accrualify?**
+
+* Once you log into your account, you will be brought to your Dashboard. From there, you will be able to see your to-do items.
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/frequently-asked-questions.md]:
```suggestion
* Once you log into your account, you will be brought to your dashboard. From there, you will be able to see your to-do items.
```

File: operations/finance/spending-company-money/how-to-use-accrualify/frequently-asked-questions.md [hunk 27]
```diff
@@ -0,0 +1,21 @@
...
+
+**What are our standard payment terms?**
+
+* We pay vendors 30 days from the date of receipt of the invoice.
+
+**I'm lost and I don’t know where to start, please help!**
+
+* No worries! Feel free to ping the [Purchases channel](https://community.mattermost.com/private-core/channels/purchases) for any purchasing related questions. The Finance team is here to help and will make sure you feel supported every step of the way!
```
Comment (cwarnermm) [inline operations/finance/spending-company-money/how-to-use-accrualify/frequently-asked-questions.md]:
```suggestion
* No worries! Feel free to ping the [Purchases channel](https://community.mattermost.com/private-core/channels/purchases) for any purchase-related questions. The Finance team is here to help and will make sure you feel supported every step of the way!
```

**Review by cwarnermm — COMMENTED** [review]
I've offered minor suggestions inline to simplify and add consistency to the user workflows via Accrualify. 

Please note that the number of suggestions provided is **by no means** an indicator of the quality of this document. It's a very good read! It's straightforward, covers common scenarios, and provides an appropriate level of detail for target audiences.

**Comment by ialorro** [comment]
Great edits, Carrie! Thank you very much!

**Review by TQuock — DISMISSED** [review]
@cwarnermm - thanks for the review of this doc. Much appreciate the revisions. It's more consistent with the tone of the handbook.

**Comment by justinegeffen** [comment]
Thanks, @ialorro! This is awesome. One last thing: Where would you like this to be placed in the LHS nav? The SUMMARY.md file is the only remaining update that's needed.

**Comment by TQuock** [comment]
@justinegeffen - can this be merged?

**Comment by justinegeffen** [comment]
@TQuock, @ialorro: I've updated the SUMMARY.md file; let me know if the organization works for you and then we're good to merge. :)

**Review by justinegeffen — APPROVED** [review]

**Comment by TQuock** [comment]
It all looks great @justinegeffen!

**Pull request merged**